pkgs/tools/virtualization/nixos-container/default.nix

```diff
@@ -0,0 +1,17 @@
+{ pkgs, ... }:
```

Comment:
Thanks :-) However, the function signature still doesn't say something like: `{ substituteAll, perl, perlPackages, shadow, utillinux }:` and pulls out the whole `pkgs` attrset instead of stating dependencies explicitly, see http://nixos.org/nixpkgs/manual/#sec-syntax (`Functions should list their expected arguments as precisely as possible. ...`).